common/supplemental/units.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE supplementalData SYSTEM "../../common/dtd/ldmlSupplemental.dtd">
+<!--
+Copyright © 1991-2013 Unicode, Inc.
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+For terms of use, see http://www.unicode.org/copyright.html
+-->
+
+<supplementalData>
+	<version number="$Revision$"/>
+	<unitConstants>
+		<unitConstant constant="ft2m" value="0.3048"/>
```

Comment:
Using '2' to mean "To" here seems to invite confusion with 2 as an exponent, especially in constructs like "cup2m3"; why not just spell out "To"?

Reply:
changing to use -to- instead of '2'.